lib/Views/UserInterface.jsx

```diff
@@ -18,55 +18,42 @@ import CelebrationStyles from "./Anniversary/celebration.scss";
 
 import "./global.scss";
 
-function loadAugmentedVirtuality(callback) {
-  require.ensure(
-    "terriajs/lib/ReactViews/Map/Navigation/AugmentedVirtualityTool",
-    () => {
-      const AugmentedVirtualityTool = require("terriajs/lib/ReactViews/Map/Navigation/AugmentedVirtualityTool");
-      callback(AugmentedVirtualityTool);
-    },
-    "AugmentedVirtuality"
-  );
-}
+// function loadAugmentedVirtuality(callback) {
+//   require.ensure(
+//     "terriajs/lib/ReactViews/Map/Navigation/AugmentedVirtualityTool",
+//     () => {
+//       const AugmentedVirtualityTool = require("terriajs/lib/ReactViews/Map/Navigation/AugmentedVirtualityTool");
+//       callback(AugmentedVirtualityTool);
+//     },
+//     "AugmentedVirtuality"
+//   );
+// }
 
-function isBrowserSupportedAV() {
-  return /Android|iPhone|iPad/i.test(navigator.userAgent);
-}
+// function isBrowserSupportedAV() {
+//   return /Android|iPhone|iPad/i.test(navigator.userAgent);
+// }
 
 export default function UserInterface(props) {
   return (
-    <>
-      <StandardUserInterface {...props} version={version}>
-        <Menu>
-          <RelatedMaps viewState={props.viewState} />
-          <MenuItem caption="About" href="about.html" key="about-link" />
-        </Menu>
-        <Nav>
-          <MeasureTool terria={props.viewState.terria} key="measure-tool" />
-          <button
-            title="Check out the new updates on NationalMap"
-            className={CelebrationStyles.toggleCelebration}
-            onClick={() => {
-              props.viewState.showCelebration = true;
-            }}
-          >
-            <HatIcon role="presentation" aria-hidden="true" />
-          </button>
-        </Nav>
-        <ExperimentalMenu>
-          <If condition={isBrowserSupportedAV()}>
-            <SplitPoint
-              loadComponent={loadAugmentedVirtuality}
-              viewState={props.viewState}
-              terria={props.viewState.terria}
-              experimentalWarning={true}
-            />
-          </If>
-        </ExperimentalMenu>
-      </StandardUserInterface>
-
-      <Celebration viewState={props.viewState} key="celebration" />
-    </>
+    <StandardUserInterface {...props} version={version}>
+      <MenuLeft>
+        <MenuItem caption="About" href="about.html" key="about-link" />
+        <RelatedMaps viewState={props.viewState} />
+      </MenuLeft>
+      <Nav>
+        <MeasureTool terria={props.viewState.terria} key="measure-tool" />
+      </Nav>
+      <ExperimentalMenu>
+        {/* <If condition={isBrowserSupportedAV()}>
+          <SplitPoint
+            loadComponent={loadAugmentedVirtuality}
+            viewState={props.viewState}
+            terria={props.viewState.terria}
+            experimentalWarning={true}
+          />
+        </If> */}
```

Comment:
I think we should test and if possible enable AR. Also something to check with @AnaBelgun

Reply:
yes, AR should be available in mobile. Are you referring to something else (i.e available in desktop as well?)

Reply:
I think it is disabled in terriamap#main too, so we need to enable it there and test.

https://github.com/TerriaJS/TerriaMap/blob/main/lib/Views/UserInterface.jsx#L39-L46